src/main/resources/frontend/src/views/StudentView/SubmissionInfo.vue

```diff
@@ -32,6 +49,22 @@ const { submission } = defineProps<{
       <span v-else>failed <i class="fa-solid fa-circle-xmark" style="color: red"/></span>
     </p>
 
+    <div v-if="useAuthStore().user?.role == 'ADMIN' && commitVerificationFailed(submission)">
+      <InfoPanel id="approveSubmission">
+        <h4>Approve Blocked Submission</h4>
+        <p>This submission was blocked because it did not meet the git commit requirements.</p>
+        <p>Meet with the student and explain the importance of frequent and consistent commits.</p>
+        <p>You may, at your discretion, deduct 10% if it looks like the student is not learning the value/habit of repeated commits</p>
```

Comment:
I would prefer to adjust the language here. The normal flow is to allow the penalty to be deducted, but sometimes the situation warrants skipping the penalty. 

Waiving the penalty should NOT be standard practice. The algorithm was built robustly; there won't be many mistakes in the grading that TA's need to waive the penalties. There seriously won't be a significant amount of students who need the penalty waived, and I'm afraid that TA's will use their digression to leniently and undercut the importance of git commits by waiving penalties too frequently.

What if we only allowed TA's to approve (with a penalty), and gave only professors the ability to waive the penalty? 

## Proposed language:
_The grading rubric defines a 10% penalty for students who fail to meet the commit verification penalty. While it's important support this policy, there are some special cases that warrant waiving the penalty. If you believe this student was wrongly blocked from submitting, and the circumstances are consistent with the latest direction from the professors, you may waive the standard penalty._
_1. Approve_
_2. Approve without penalty_

Reply:
@prattnj @jerodw @leesjensen @rodham  As the professors and head TA, how do you feel about this discussion? Would you be OK with reserving the ability to waive penalties for professors only?

Reply:
The wording I used here comes specifically from Professor Jensen's message in the Slack when the decision was made to do this. I'm down for whatever language the professors decide![image](https://github.com/softwareconstruction240/autograder/assets/112426674/5e8a82e8-9025-4dd8-b5f1-4d0763603c0a)

Reply:
Reposted here for convenience
![image](https://github.com/softwareconstruction240/autograder/assets/19808440/a1e0f9a3-e7ab-44a3-ad0c-331e0eeeb2a8)

Thanks for providing that screenshot. You definitely have a good point, and I still think TA's should generally apply the penalty more often than they waive it.